mm2src/coins/lightning/ln_events.rs

```diff
@@ -222,28 +299,114 @@ impl LightningEventHandler {
             "Handling PaymentSent event for payment_hash: {}",
             hex::encode(payment_hash.0)
         );
-        if let Some(payment) = self.outbound_payments.lock().get_mut(&payment_hash) {
-            payment.preimage = Some(payment_preimage);
-            payment.status = HTLCStatus::Succeeded;
-            payment.fee_paid_msat = fee_paid_msat;
-            log::info!(
-                "Successfully sent payment of {} millisatoshis with payment hash {}",
-                payment.amt_msat.unwrap_or_default(),
-                hex::encode(payment_hash.0)
-            );
+        let persister = self.persister.clone();
+        spawn(async move {
+            if let Ok(Some(mut payment_info)) = persister
+                .get_payment_from_sql(payment_hash)
+                .await
+                .error_log_passthrough()
```

Comment:
you use it here :D

Reply:
The result is used here, so no clippy errors if I used `error_log_passthrough()` here.